.appveyor.yml

```diff
@@ -14,14 +14,17 @@ build:
 
 environment:
     matrix:
-      - CMAKE_GENERATOR: "Ninja"
-      - CMAKE_GENERATOR: "MinGW Makefiles"
```

Comment:
Not because I didn't want to, I simply couldn't get it to work. If MinGW is in the path, then Ninja gets messed up in the head and tries linking MinGW libraries to objects compiled by MSVC, then If I try to conditionaly remove it from `PATH`, then `cmd` gets high on acid and starts complaining about some "google cloud" thingy - somehow setting PATH within an `if` statement doesn't work on appveyor, I tried quoting the hell out of it, but to no avail.

Since MinGW is essentially GCC on windows, I felt there was no added value in testing it.